tests/test_datasets.py

```diff
@@ -97,7 +97,7 @@ def test_dataset(self) -> None:
                 local_cache_path_prefix=tmpdir,
                 columns_to_load=None,
                 storage=fake_s3_storage,
```

Comment:
I noticed these unit tests did not use the mock `fake_s3_storage` as the argument value to the `storage` parameter. I checked the code and it turns out that it doesn't matter to the correctness of the test (the test passes either way). However, as a best practice I fixed these lines.